index.js

```diff
@@ -2,32 +2,65 @@ const core = require('@actions/core');
 const exec = require('@actions/exec');
 const github = require('@actions/github');
 
-const ARGS = [
+const ALL_ARGS = [
     'run', '--all-files', '--show-diff-on-failure', '--color=always'
 ];
 
+const CHANGED_ARGS = [
+    'run', '--show-diff-on-failure', '--color=always', '--files'
+];
+
+
 function addToken(url, token) {
     return url.replace(/^https:\/\//, `https://x-access-token:${token}@`)
 }
 
+async function getChangedFiles() {
+
+    let changedFiles = '';
+    const baseRef = core.getInput('base');
+
+    const options = {};
+    options.listeners = {
+        stdout: (data) => {
+            changedFiles += data.toString();
+        },
+    };
+
+    await exec.exec('git', ['diff', baseRef, '--name-only'], options);
+
+    return changedFiles;
+}
+
 async function main() {
+    var ARGS
```

Comment:
this isn't a constant so it shouldn't be named as such, also you're missing a semicolon